sdk/storage/storage-blob-changefeed/test/blobchangefeedclient.spec.ts

```diff
@@ -2,619 +2,667 @@
 // Licensed under the MIT License.
 
 import { isPlaybackMode, Recorder, env } from "@azure-tools/test-recorder";
-import { recorderEnvSetup, getBlobChangeFeedClient, streamToString, uriSanitizers } from "./utils";
-import type { BlobChangeFeedEvent, BlobChangeFeedEventPage } from "../src";
-import { BlobChangeFeedClient } from "../src";
-import { assert } from "@azure-tools/test-utils";
+import {
+  recorderEnvSetup,
+  getBlobChangeFeedClient,
+  streamToString,
+  uriSanitizers,
+} from "./utils/index.js";
+import type { BlobChangeFeedEvent, BlobChangeFeedEventPage } from "../src/index.js";
+import { BlobChangeFeedClient } from "../src/index.js";
 import type { BlobServiceClient, RequestPolicy } from "@azure/storage-blob";
-import { SDK_VERSION } from "../src/utils/constants";
-import * as fs from "fs";
-import * as path from "path";
-
-import type { Context } from "mocha";
-import { rawEventToBlobChangeFeedEvent } from "../src/utils/utils.common";
+import { SDK_VERSION } from "../src/utils/constants.js";
+import * as fs from "node:fs";
+import * as path from "node:path";
+import { rawEventToBlobChangeFeedEvent } from "../src/utils/utils.common.js";
 import type { RestError } from "@azure/core-rest-pipeline";
 import { createHttpHeaders } from "@azure/core-rest-pipeline";
 import { toHttpHeadersLike } from "@azure/core-http-compat";
+import { describe, it, assert, expect, beforeEach, afterEach } from "vitest";
+import { toSupportTracing } from "@azure-tools/test-utils-vitest";
+import type { OperationOptions } from "@azure/core-client";
+
+expect.extend({ toSupportTracing });
 
 const timeoutForLargeFileUploadingTest = 20 * 60 * 1000;
 
-describe("BlobChangeFeedClient", async () => {
-  let recorder: Recorder;
-  let changeFeedClient: BlobChangeFeedClient;
+describe(
+  "BlobChangeFeedClient",
+  { skip: true },
+  // TODO: Enable { skip: env.CHANGE_FEED_ENABLED !== "1" && !isPlaybackMode() },
+  async () => {
+    let recorder: Recorder;
+    let changeFeedClient: BlobChangeFeedClient;
+
+    beforeEach(async (ctx) => {
+      recorder = new Recorder(ctx);
+      await recorder.start(recorderEnvSetup);
+      // make sure we add the sanitizers on playback for SAS strings
+      await recorder.addSanitizers({ uriSanitizers }, ["record", "playback"]);
+      changeFeedClient = getBlobChangeFeedClient(recorder);
+    });
 
-  before(async function (this: Context) {
```

Comment:
Anyway to get this back?

Reply:
This should be in the file and wasn't deleted, but instead moved around.  I think the git diff is a little weird for this file.